projects/com.oracle.truffle.llvm.parser.impl/src/com/oracle/truffle/llvm/parser/impl/LLVMParserAsserts.java

```diff
@@ -40,4 +40,10 @@
         return objects;
     }
 
+    public static void assertNotNull(Object object) {
+        if (object == null) {
+            throw new AssertionError();
+        }
+    }
```

Comment:
I would like to keep all asserts in this class. This would allow us, e.g., to later replace the `AssertionError` with an `assert`. I also think that it is more readable to use `LLVMParserAsserts`.